lib/mockResponse.js

```diff
@@ -378,14 +378,14 @@ function createResponse(options) {
     };
     
     /**
-     *  Function: getEndParameters
+     *  Function: getEndArguments
```

Comment:
@eugef I extracted the function and renamed it first getEndParameters and now getEndArguments. What do you think? :)